universities/indiana.md

```diff
@@ -0,0 +1,13 @@
+# Indiana University
+
+- *OSPO*: No
+- *Link*: [Open Source Policy & Procedures](https://indiana-university.github.io/)
+
+# Overview
+
+IU's Open Source Program provides comprehensive guidelines and administrative support for open source initiatives. It encourages the establishment and publishing of open source code repositories under specific conditions, ensuring that projects have utility beyond the university and represent innovative contributions. The program mandates that all open source repositories be approved by an authorized individual at the director level or higher within the associated unit.
```

Comment:
Why isn't that an OSPO? Who runs this?

Reply:
As far as I can tell an office within the university hasn't been formed. There is an email (opensource@iu.edu) and some guidelines https://indiana-university.github.io/ but no actual organizational cut-out as far as I can tell or even an individual responsible for this work. I may have missed something though.

Reply:
Cool. Should we add that information here? What you've just written is useful to someone who is viewing this page - I think this is the kind of stuff we want to have in the map.